tests/src/JIT/SIMD/CircleInConvex.cs

```diff
@@ -259,8 +259,9 @@ static int Main(string[] args)
             float r;
             FindCircle(points, out O, out r);
 
-            float expRes = 7.565624E7F;
-            if (Math.Abs(r - expRes) > EPS)
+            float expRes = 75656240.0F;
+            float ulp    =        8.0F;
```

Comment:
The value is 1 ULP (unit in the last place) for the expected result value (i.e. the difference between the expected result, represented as float, and the next representable value), computed manually. It could be computed from the value, but would require several steps, including reinterpreting the bits as an int, so it seemed best to just use a "magic" constant, as is done for `expRes`.